### PR TITLE
Clean up init and remove debug from kernel cmdline

### PR DESCRIPTION
**Pull request by mariogrip**
This cleans up init, and removes services that is not needed. It also removes debug from kernel cmdline.

### CODE REVIEW

**Comment by mariogrip** [comment]
[Stabilize the device](https://trello.com/c/eZc8Kwrw/17-stabilize-the-device)

File: rootdir/root/init.qcom.rc [hunk 1]
```diff
@@ -460,7 +424,6 @@ service qmuxd /system/bin/qmuxd
 
 service netmgrd /system/bin/netmgrd
     class main
-    disabled
```
Comment (peat-psuwit) [inline rootdir/root/init.qcom.rc:463]: init.class_main.sh do start netmgrd depends on ro.use_data_netmgrd property. We should investigate further why this script isn't run.

File: rootdir/root/init.qcom.rc [hunk 2]
```diff
@@ -627,33 +522,14 @@ service qcom-post-boot /system/bin/sh /system/etc/init.qcom.post_boot.sh
...
-    class late_start
-    user system
-    group system radio
-
 service hostapd /system/bin/hostapd -dd /data/hostapd/hostapd.conf
     class late_start
     user root
     group root
     oneshot
-    disabled
```
Comment (peat-psuwit) [inline rootdir/root/init.qcom.rc:656]: I don't think this should start. I don't even think this service should be in the init file at all. It should be on Ubuntu part.

**Review by peat-psuwit — CHANGES_REQUESTED** [review]
I think we should investigate further why the script doesn't start netmgrd. Besides, you shouldn't include the merge commit in the pull request.

File: rootdir/root/init.qcom.rc (hunk 1 above)
Comment (ghost) [reply rootdir/root/init.qcom.rc:463]: I agree, the same goes for ril-daemon and qmux services, they should also be started from main. It looks like init.class_main.sh isn't processed at all

File: rootdir/root/init.qcom.rc [hunk 3]
```diff
@@ -404,7 +404,6 @@ service irsc_util /system/bin/irsc_util "/etc/sec_config"
 service rmt_storage /system/bin/rmt_storage
     class core
     user root
-    disabled
```
Comment (ghost) [inline rootdir/root/init.qcom.rc:407]: This is started below on property:ro.boot.emmc=true?

File: rootdir/root/init.qcom.rc (hunk 1 above)
Comment (peat-psuwit) [reply rootdir/root/init.qcom.rc:463]: Ok. Just noticed that init.environ.rc, which set PATH variable amongst other things, is blamed for making android container crashes and is commented out of init.rc. This makes many of shell command (for example, start command which starts a service in the Android container) not working at all. I'll find out why init.environ.rc crashes the container and re-adding it to init.rc.

Comment (ghost) [reply rootdir/root/init.qcom.rc:463]: I think I found it. Android cannot find getprop. Adding PATH=/system/bin:$PATH to init.class_main.sh makes it work.

Comment (peat-psuwit) [reply rootdir/root/init.qcom.rc:463]: The problem is bigger than that. I'll take a look at why init.environ.rc makes Android container crash at all. Qcom's init system is driven by a number of shell script, and patching all of them is not a solution. Besides, there's probably more things that rely on the correct environment variable being set.

**Pull request closed without merging**